app/src/main/java/com/example/ai_stuttering_assistant/MainActivity.kt

```diff
@@ -120,24 +140,26 @@ fun ButtonExample(context: Context, onClick: () -> Unit) {
 
 @Composable
 fun Interface(context: Context) {
-    Column(
-        modifier = Modifier
-            .padding(all = 8.dp)
-            .fillMaxWidth(),
+    LazyColumn(
+        contentPadding = WindowInsets.systemBars.asPaddingValues(),
+        modifier = Modifier.fillMaxWidth(),
         horizontalAlignment = Alignment.CenterHorizontally
     ) {
-        Image(
-            painter = painterResource(id = R.drawable.misato),
-            contentDescription = "Misato Katsuragi",
-            modifier = Modifier
-                .size(500.dp)
-        )
-        Spacer(modifier = Modifier.height(16.dp))
-        Text("Misato Katsuragi", fontSize = 24.sp)
-        Spacer(modifier = Modifier.height(100.dp))
-        ButtonExample(
-            context = context,
-            onClick = { Log.d("talk-button", "BUTTON CLICKED") }
-        )
+        item {
+            Image(
+                painter = painterResource(id = R.drawable.misato),
+                contentDescription = "Misato Katsuragi",
+                modifier = Modifier
+                    .size(500.dp)
+            )
+            Spacer(modifier = Modifier.height(16.dp))
+            Text("Misato Katsuragi", fontSize = 24.sp)
+            Spacer(modifier = Modifier.height(100.dp))
+            ButtonExample(
+                context = context,
+                onClick = { Log.d("talk-button", "BUTTON CLICKED") }
+            )
+        }
```

Comment:
`LazyColumn` allows for native scrolling and properly applies padding for overlaps in the navigation and status bars. All content has been moved into one `item` container as it is not much, it can all be loaded at once.